src/usdb_syncer/gui/settings_dialog.py

```diff
@@ -42,6 +45,62 @@ def __init__(self, parent: QWidget, song: UsdbSong | None) -> None:
         self.label_video_embed_artwork.setVisible(False)
         self.checkBox_video_embed_artwork.setVisible(False)
         self.groupBox_reencode_video.setVisible(False)
+        if sys.platform != "win32":
+            self.groupBox_vocaluxe.setVisible(False)
+        self.pushButton_browse_karedi.clicked.connect(
+            lambda: self._set_location(settings.SupportedApps.KAREDI)
```

Comment:
This saves the setting immediately. All other settings are only saved once the user presses Ok.

Reply:
So just deleted `settings.set_app_path(app, path)`?